File: exercises/small_problems/easy6/delete_vowels.rb
# START
# INPUT array of strings
# FOR EACH string in array
#     REMOVE remove vowels
# OUTPUT new array sans vowels 
# END

def remove_vowels(arr)
  arr.map! do |str|
    str.delete("AEIOUaeiou")
  end
end

p remove_vowels(%w(abcdefghijklmnopqrstuvwxyz)) #== %w(bcdfghjklmnpqrstvwxyz)
p remove_vowels(%w(green YELLOW black white)) #== %w(grn YLLW blck wht)
p remove_vowels(%w(ABC AEIOU XYZ)) #== ['BC', '', 'XYZ']

File: exercises/jsSmallProblems/stringProcessing/deleteVowels.js
const removeVowels = (array) => (
  array.map((string) => string.match(/[^aeiou]*/gi).join(''))
);


console.log(removeVowels(['abcdefghijklmnopqrstuvwxyz'])); // ["bcdfghjklmnpqrstvwxyz"]
console.log(removeVowels(['green', 'YELLOW', 'black', 'white'])); // ["grn", "YLLW", "blck", "wht"]
console.log(removeVowels(['ABC', 'AEIOU', 'XYZ'])); // ["BC", "", "XYZ"]
